lang/src/test/kotlin/org/partiql/lang/planner/transforms/AstToLogicalVisitorTransformTests.kt

```diff
@@ -877,21 +877,20 @@ class AstToLogicalVisitorTransformTests {
         )
     }
```

Comment:
This file's changeset restores the state of AstToLogicalVisitorTransformTests.kt to what it was before PR #992.  While figuring out JUnit5 here, I now saw another method of getting the identification of the failed test that I was after in PR992: re-run the suite with the failing test in IntelliJ -- it auto-numbers the cases of parameterized test.  It is not as good as the searchability of the id in PR992 (say, getting to the source of the offending parameters by clicking on the failure in the UI would be great), but is close enough and avoids the undeniable extra cruft of user-created IDs. 

Having said this, but looking at what happens in StatementRedactorTest in this PR (~80 test parameters with subtly different arguments), I am maybe getting by PR992 opinion back...